src/services/importService.ts

```diff
@@ -70,32 +74,57 @@ export default class ImportService implements IImportService {
      */
     public async generateAssets(v1Project: IFileInfo, v2Project: IProject): Promise<IAssetMetadata[]> {
         let originalProject: IV1Project;
-        const generatedAssetMetadata: IAssetMetadata[] = [];
-        let assetState: AssetState;
+        let generatedAssetMetadata: IAssetMetadata[] = [];
         const assetService = new AssetService(v2Project);
 
         originalProject = JSON.parse(v1Project.content as string);
 
-        for (const frameName in originalProject.frames) {
-            if (originalProject.frames.hasOwnProperty(frameName)) {
-                const frameRegions = originalProject.frames[frameName];
-                const asset = AssetService.createAssetFromFilePath(
-                    `${v1Project.file.path.replace(/[^\/]*$/, "")}${frameName}`);
-                const populatedMetadata = await assetService.getAssetMetadata(asset).then((metadata) => {
-                    assetState = originalProject.visitedFrames.indexOf(frameName) > -1 && frameRegions.length > 0
-                        ? AssetState.Tagged : (originalProject.visitedFrames.indexOf(frameName) > -1
-                        ? AssetState.Visited : AssetState.NotVisited);
-                    const taggedMetadata = this.addRegions(metadata, frameRegions);
-                    taggedMetadata.asset.state = assetState;
-                    taggedMetadata.asset.path = `file:${taggedMetadata.asset.path}`;
-                    return taggedMetadata;
-                });
-                generatedAssetMetadata.push(populatedMetadata);
-            }
+        const pathParts = v1Project.file.path.split(/[\\\/]/);
```

Comment:
I feel like we're duplicating a lot of the logic that already exists within the `AssetService`.  Please refactor to re-use as much as you can.